apps/docs/docs/core/core.md

```diff
@@ -346,3 +346,32 @@ const parentContract = c.router(
   }
 );
 ```
+
+### Validate Response
```

Comment:
This should be part of the `core/fetch` page instead

Reply:
Done b8fa3e8